actions.yaml

```diff
@@ -0,0 +1,7 @@
+cis-benchmark:
+  description: CIS Kubernetes Benchmark
```

Comment:
Can we be more descriptive here? I think this should describe what this action will do.

Reply:
Traditionally, we've steered away from long action descriptions to keep the `juju actions $application` output tidy. I'll make it a bit more verbose, e.g.:

```
Run the CIS Kubernetes Benchmark against snap-based components.
```

But I'd prefer to write up a full description in our k8s docs.